.mailmap

```diff
@@ -0,0 +1,2 @@
+Jean-Noël Avila <jn.avila@free.fr>
+Scott Chacon <schacon@gmail.com>
```

Comment:
I wasn't sure if this file should be tweaked to contain polish translation maintainer, both or just removed totally.